worlds/messenger/docs/en_The Messenger.md

```diff
@@ -49,28 +49,28 @@ for it. The groups you can use for The Messenger are:
 ## Other changes
 
 * The player can return to the Tower of Time HQ at any point by selecting the button from the options menu
-  * This can cause issues if used at specific times. Current known:
-    * During Boss fights
-    * After Courage Note collection (Corrupted Future chase)
-      * This is currently an expected action in logic. If you do need to teleport during this chase sequence, it
-        is recommended to quit to title and reload the save
+    * This can cause issues if used at specific times. Current known:
```

Comment:
"Current known" feel weird, might not be correct in terms of syntax? Might be better to change with "Currently known issues". 
Current version is understandable, so a strong suggestion but not a required one.